.circleci/config.yml

```diff
@@ -161,49 +203,79 @@ jobs:
       - store_artifacts:
           path: test-results
 
-  unpackaged-sanity:
+
+  linux-tarball-test:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/repo
+
+      # We'll want to use the LTS version of node
+      - update_node:
+          os: linux
+
+      # Set up the plugin
+      - run:
+          name: "Install SFDX and tarball"
+          command: |
+            npm install -g sfdx-cli
+            # We need to determine the name of the tarball
+            SFDC_TARBALL=$(ls | grep salesforce*.tgz)
+            # Pipe in a 'y' to simulate agreeing to install an unsigned plugin. Use a URI of the file's full path.
+            echo y | sfdx plugins:install file://`pwd`/${SFDC_TARBALL}
```

Comment:
* Can `pwd` be moved up to initialize the same way that `SFDC_TARBALL` is? 
* It's a best practice to quote the `file://pwd/${SFDC_TARBALL}` in case the directory ever has spaces